R/utils-keep_range_slider_updated.r

```diff
@@ -25,8 +26,14 @@ keep_range_slider_updated <- function(session,
     step <- NULL
 
     if (isTRUE(is_density)) {
-      minmax <- c(0, round(max(stats::density(stats::na.omit(ANL[[varname]]))$y) * 1.5, 5))
-      step <- round(max(stats::density(stats::na.omit(ANL[[varname]]))$y) / 100, 5)
+      treatname <- input[[id_trt_group]]
+      ANL_split <- ANL %>% split(f = factor(paste0(ANL[['AVISITCD']], ANL[[treatname]])))
+      density_maxes <- lapply(ANL_split, function(x){
+        max(stats::density(stats::na.omit(x[[varname]]))$y)
+      })
+      dmax <- max(unlist(density_maxes))
+      minmax <- c(0, round(dmax * 1.2, 5))
```

Comment:
I think we can even try without this
Previously it was multiplied by `1.5` because the behavior of curves being outside of plot ranges was known, and this was a way to fix it (for some cases). Now we properly calculate the max, so multiplication is not that crucial
```suggestion
      minmax <- c(0, round(dmax, 5))
```

Reply:
I tried without the multiplication and one graph has the curve closer to the top.
![image](https://github.com/user-attachments/assets/ce923b82-b200-4c6d-ac50-088d1e5801a4)

I personally like it better with the multiplication so I can see the y-axis tick, but I wonder if there's another reason that this multiplication was added.

I would prefer to leave it as is for now.